src/config.hpp

```diff
@@ -51,7 +51,7 @@ bool Main::Config::ExecuteConfigCommands(Ui::Command & handler)
       if (xdg_config_dir != NULL)
          configFile = std::string(xdg_config_dir).append("/vimpc/vimpcrc");
       else
-         configFile = std::string(home_dir).append("/.vimpcrc");
```

Comment:
I would say the program should look for `~/.vimpcrc` if `$XDG_CONFIG_HOME/vimpc/vimpcrc` or `~/.config/vimpc/` is not found. IIRC `mpd` and many other programs behave in this way.

Reply:
Agreed, that would be the ideal functionality. If implemented I will merge

Reply:
I've updated the logic accordingly.